test_runner: pass abort signal to test (#43554)
nodejs/node: doc/api/test.md

  * `only` {boolean} If truthy, and the test context is configured to run
    `only` tests, then this test will be run. Otherwise, the test is skipped.
    **Default:** `false`.

>   * `signal` {AbortSignal} allows aborting an in-progress test

Reviewer: ```suggestion
  * `signal` {AbortSignal} Allows aborting an in-progress test.
```

});
```

### `context.signal`

<!-- YAML
added: REPLACEME
-->

> this is an <AbortSignal> used to signal when the test has been aborted.

Reviewer: ```suggestion
* <AbortSignal> Can be used to abort test subtasks when the test has been aborted.
```

});
```

## Class: `SuiteContext`

<!-- YAML
added: REPLACEME
-->

An instance of `SuiteContext` is passed to each suite function in order to
interact with the test runner. However, the `SuiteContext` constructor is not
exposed as part of the API.

### `context.signal`

<!-- YAML
added: REPLACEME
-->

> this is an <AbortSignal> used to signal when the test has been aborted.

Reviewer: ```suggestion
* <AbortSignal> Can be used to abort test subtasks when the test has been aborted.
```
